SimpleWAWS/Authentication/AuthUtilities.cs

```diff
@@ -1,17 +1,24 @@
 ﻿using System;
-using System.Collections.Generic;
 using System.IO;
-using System.Linq;
 using System.Net;
-using System.Web;
 
 namespace SimpleWAWS.Authentication
 {
     public static class AuthUtilities
     {
-        public static string GetContentFromUrl(string url)
+        public static string GetContentFromUrl(string url, string method="GET", bool jsonAccept = false, bool addGitHubHeaders = false, string AuthorizationHeader ="")
```

Comment:
Actually I'd create another method specifically for github and refactor these 2 methods since it's a bit confusing now.